frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx

```diff
@@ -54,25 +54,25 @@ const DashboardInfoSidebar = ({
   const handleDescriptionChange = useCallback(
     (description: string) => {
       setDashboardAttribute("description", description);
-      saveDashboardAndCards(dashboard.id);
+      saveDashboardAndCards(true);
```

Comment:
There are 2 changes here
1. Fix the dashboard action API since it previously didn't expect any parameter at all.
2. Introduce the flag to specify that we want to preserve parameter values.
    This is for actions that don't touch parameters at all which are modifying a dashboard description, cache TTL, and auto-apply filters in this file